.gitmodules

```diff
@@ -13,3 +13,9 @@
 [submodule "contracts/lib/canonical-weth"]
 	path = contracts/lib/canonical-weth
 	url = https://github.com/Snowfork/canonical-weth
+[submodule "lodestar"]
```

Comment:
Why are we making this a submodule?

Reply:
Last time when I upgrade lodestar to v1.11 met an issue which breaks the api of dumping beacon state we rely on to generate the ancestry proof,  made a fix in https://github.com/Snowfork/lodestar/pull/1 to resolve it. 

Since it's already fixed in v1.12 and just tested that e2e smoke tests work as expected so yes we do not need the fork anymore.

But that's somehow explains the reason why I prefer to fork the upstream repos like geth and lodestar, it allows us to test with the most recent version and make any fix or hack as we want, like the [hack](https://github.com/Snowfork/snowbridge/blob/cce771451a53260523bc5da6efc46c60ebb51f10/web/packages/test/scripts/build-binary.sh#L41-L49) for a fast block time to speedup the e2e test.